### ISSUE TITLE
Looking for a valid tag sometimes gets "Tag not found" error.

### ISSUE THREAD

**Issue by thegavaguy** [gwsw/less#146]
I often use Exuberant Ctags to create a `tags` file for a project and then use `less`, as well as my editor, to poke around source code.  Sometimes, `less` gives the error "Tag not found" when searching for a tag's location, even though I know for certain that the `tags` file is up to date and other tools locate the tag just fine.

Reproducing the problem can be done easily using the source code for `less` itself.  From the directory with the code for `less` issue the following commands:

```
$ ctags *.[ch]
$ less -t anchor
Tag not found  (press RETURN)
```

Note that most of the tags are correctly found.  Picking one at random, the command `less -t cmdtable` goes to the file decode.c and shows `(tag 1 of 2)`.  Typing `t` goes to the file lesskey.c for the second location for that tag and typing `T` returns to the previous location for that tag, as expected.

Perhaps there are not a lot of `less` users who avail themselves of its ability to quickly jump to the definition of some variable, function, macro, etc., but I find it very handy.  So, being able to locate all valid tags is important to me.  Many thanks for incorporating this functionality into `less`.

I have looked into this issue and will post my suggested solution in a pull request shortly.

**Comment by polluks** [comment]
Maybe it's rather a ctags issue?
````
$ grep anchor tags
add_hilite	search.c	/^add_hilite(anchor, hl)$/
clr_hlist	search.c	/^clr_hlist(anchor)$/
hlist_rotate_left	search.c	/^hlist_rotate_left(anchor, n)$/
hlist_rotate_right	search.c	/^hlist_rotate_right(anchor, n)$/
````

**Comment by gwsw** [comment]
Thanks, merged in 97ecc8cb5c3ee5a8bad98a6cdb54aecda7bd961c.
